framework/test/mocha/unit/modules/chain/logic/test_data/multisignature.js

```diff
@@ -38,7 +35,10 @@ const senderHash = crypto
 	.createHash('sha256')
 	.update(validSender.passphrase, 'utf8')
 	.digest();
-const senderKeypair = ed.makeKeypair(senderHash);
+
+const senderKeypair = getPrivateAndPublicKeyBytesFromPassphrase(
```

Comment:
this will be `senderKeypair.publicKeyBytes` but it seems below didnt change is that ok?

Reply:
Just double checked and it seems these files inside `test_data/` folder are not used. I will remove them all.